src/bionics_ui.cpp

```diff
@@ -121,6 +124,48 @@ std::string build_bionic_powerdesc_string( bionic const &bio )
     return power_desc.str();
 }
 
+void show_bionics_tabs( WINDOW *win, size_t active_num, size_t passive_num,
+                        bionic_tab_mode current_mode )
+{
+    werase( win );
+
+    int width = getmaxx( win );
+    mvwhline( win, 2, 0, LINE_OXOX, width );
+
+    std::ostringstream active_tab_name;
+    active_tab_name << _( "ACTIVE" ) << num_in_parentheses( active_num );
+    std::ostringstream passive_tab_name;
+    passive_tab_name << _( "PASSIVE" ) << num_in_parentheses( passive_num );
+
+    const int tab_step = 3;
+    int tab_x = 1;
+    draw_tab( win, tab_x, active_tab_name.str(), current_mode == TAB_ACTIVE );
+    tab_x += tab_step + utf8_width( active_tab_name.str() );
+    draw_tab( win, tab_x, passive_tab_name.str(), current_mode == TAB_PASSIVE );
```

Comment:
This function has a bit of duplication which would make adding a new tab a bit harder than it should be. I'd rewrite it to be like this (a snippet of pseudocode just for example):
```c++
werase( win );
draw_bionic_tab( _( "ACTIVE" ), ... );
draw_bionic_tab( _( "PASSIVE" ), ... );
wrefresh( win );
```
This would also make the (a bit awkward) `num_in_parentheses()` lambda redundant. IMHO the best solution here is turning tabs into classes and using polymorphism to achieve variability. The current approach is a bit procedural (not necessarily bad, but making extension hareder).